docs/howto/stack_version_support.md

```diff
@@ -0,0 +1,80 @@
+Intro...
+
+TBD: When to reconsider the version constraint.
+
+
+Support for all versions since the release of the package
+----
+Example: `>=7.16.0`.
```

Comment:
If you described this case, then you should also cover simple `^7.16.0` (packages won't be present in 8.0.0)